File: CS/SplitDescendantWithExtension/GridSplitContainerDescendant.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraGrid;
using DevExpress.XtraGrid.Views.Base;

namespace GridViewFixedRowsSplitPanel {
    public class GridSplitContainerDescendant : GridSplitContainer
    {
        protected override DevExpress.XtraEditors.Drawing.SplitContainerViewInfo CreateContainerInfo() {
            return new SplitContainerViewInfoDescendant(this);
        }
        protected override void OnSplitterPositionChanged() {
            base.OnSplitterPositionChanged();
        }
    }
}

File: VB/SplitDescendantWithExtension/GridSplitContainerDescendant.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraGrid
Imports DevExpress.XtraGrid.Views.Base

Namespace GridViewFixedRowsSplitPanel
    Public Class GridSplitContainerDescendant
        Inherits GridSplitContainer

        Protected Overrides Function CreateContainerInfo() As DevExpress.XtraEditors.Drawing.SplitContainerViewInfo
            Return New SplitContainerViewInfoDescendant(Me)
        End Function
        Protected Overrides Sub OnSplitterPositionChanged()
            MyBase.OnSplitterPositionChanged()
        End Sub
    End Class
End Namespace
